src/flask_sqlalchemy/extension.py

```diff
@@ -950,9 +950,7 @@ def _set_rel_query(self, kwargs: dict[str, t.Any]) -> None:
 
             backref[1].setdefault("query_class", self.Query)
 
-    def relationship(
-        self, *args: t.Any, **kwargs: t.Any
-    ) -> sa_orm.RelationshipProperty[t.Any]:
+    def relationship(self, *args: t.Any, **kwargs: t.Any) -> sa_orm.Relationship[t.Any]:
```

Comment:
Do we have any tests that use relationship()? I've been able to get test failures in the past for typing errors, since mypy runs on the tests.